.travis.yml

```diff
@@ -24,20 +24,28 @@ branches:
   only:
     - develop
     - master
-    - travis-firefox
-    - staging
+    - py3-update
 
 before_install:
-  - pip install virtualenv coveralls scikit-learn[alldeps] pandas
   - gem install coveralls-lcov
   - nvm install $TRAVIS_NODE_VERSION
   - npm install -g bower
   - npm install -g grunt-cli
   - npm install -g karma-cli
 
 install:
-  - pip install -r src/requirements.txt
-  - pip install --no-dependencies clustergrammer_widget
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - conda info -a
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
+  - source activate test-environment
+  # - pip install -r src/requirements.txt
+  # - pip install coveralls scikit-learn[alldeps] pandas
+  # - pip install --no-dependencies clustergrammer_widget
```

Comment:
Do you want to keep these commented lines?

Reply:
Nope. Deleted!